diff --git a/lib/Utils/validate-connection.js b/lib/Utils/validate-connection.js
index 6a65a7354be67ac7ab7b2c3c6c0802ce56b08347..17c14bd92b785888be962ca6c8b107333aaae78c 100644
--- a/lib/Utils/validate-connection.js
+++ b/lib/Utils/validate-connection.js
@@ -28,7 +28,7 @@ const getUserAgent = (config) => {
 };
 const PLATFORM_MAP = {
     'Mac OS': proto.ClientPayload.WebInfo.WebSubPlatform.DARWIN,
-    Windows: proto.ClientPayload.WebInfo.WebSubPlatform.WIN32
+    Windows: proto.ClientPayload.WebInfo.WebSubPlatform.WIN_HYBRID
 };
 const getWebInfo = (config) => {
     let webSubPlatform = proto.ClientPayload.WebInfo.WebSubPlatform.WEB_BROWSER;
